internal/cmd/root.go

```diff
@@ -34,9 +36,34 @@ func init() {
 
 	rootCmd.PersistentPostRun = func(cmd *cobra.Command, args []string) {
 		settings.PersistChanges()
+		if version == "dev" {
+			return
+		}
+		settings, err := settings.ReadSettings()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to read settings: %s\n", err)
+			os.Exit(1)
+		}
+		if settings.GetAutoupdate() == "on" && time.Now().Unix() >= settings.GetLastUpdateCheck()+int64(24*60*60) {
+			fmt.Println("Checking for updates...")
```

Comment:
Checking for updates should be pretty quick.
I'd make it silent unless we actually trigger the update.